src/show/pages/private/EditProject/view.jsx

```diff
@@ -0,0 +1,65 @@
+import { useEffect } from 'react';
+import PropTypes from 'prop-types';
+import { mock } from 'process/helpers';
+import StepWizard from 'react-step-wizard';
+import { useParams } from 'react-router-dom';
+
+import { editProject, getProject } from 'process/slices/projectSlice';
```

Comment:
Those should not be here, they should pe taken from the component s props.

Reply:
done

Reply:
done, ups